src/OSPSuite.Presentation/Presenters/Charts/ChartDisplayPresenter.cs

```diff
@@ -317,15 +311,12 @@ public virtual void ExportToExcel()
          var visibleCurves = Chart.Curves.Where(x => x.Visible).ToList();
          if (!visibleCurves.Any()) return;
 
-         var fileName = _dialogCreator.AskForFileToSave(Captions.ExportChartToExcel, Constants.Filter.EXCEL_SAVE_FILE_FILTER, Constants.DirectoryKey.REPORT);
+         var fileName = _dialogCreator.AskForFileToSave(Captions.ExportChartToExcel, Constants.Filter.EXCEL_SAVE_FILE_FILTER, Constants.DirectoryKey.REPORT, Chart.Name);
```

Comment:
Added a default file name (chart.NAme... if someone took the time to give it a name, why not use it)